buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import dev.panuszewski.gradle.pluginMarker
+
+plugins {
+    id("java-library")
+    `kotlin-dsl`
+    alias(libs.plugins.spotless)
+    alias(libs.plugins.detekt) apply false
+}
+
+repositories {
+    gradlePluginPortal()
```

Comment:
This thread is only for discussion. Did you see scenario where some plugin could only be found in `gradlePluginPortal()`, not in `mavenCentral()`? I asked for I struggled on whehter I need to use `gradlePluginPortal()` in `mongo-hibernate` project as well.

Reply:
Just following the official recommendations. I imagine any third party plugins would be on maven but any gradle specific ones might not be.